CMakeLists.txt

```diff
@@ -464,6 +464,9 @@ OCV_OPTION(WITH_ITT "Include Intel ITT support" ON
 OCV_OPTION(WITH_PROTOBUF "Enable libprotobuf" ON
   VISIBLE_IF TRUE
   VERIFY HAVE_PROTOBUF)
+OCV_OPTION(WITH_IMGCODEC_GIF "Include GIF support" ON
```

Comment:
I believe we should disable it by default at least for now to allow users test it in safe environment and our OSS-Fuzz builds.